extensions/AWSSDK.Extensions.sln

```diff
@@ -54,10 +54,26 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.Extensions.CloudFron
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.Extensions.EC2.DecryptPassword.NetFramework", "src\AWSSDK.Extensions.EC2.DecryptPassword\AWSSDK.Extensions.EC2.DecryptPassword.NetFramework.csproj", "{3EC669E6-A541-445E-B68E-0A853715E39C}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.Extensions.Logging.ILoggerAdaptor", "src\AWSSDK.Extensions.Logging.ILoggerAdaptor\AWSSDK.Extensions.Logging.ILoggerAdaptor.csproj", "{0A257E97-E0E9-43DE-8FEE-67382307725A}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AWSSDK.Extensions.Logging.Log4NetAdaptor", "src\AWSSDK.Extensions.Logging.Log4NetAdaptor\AWSSDK.Extensions.Logging.Log4NetAdaptor.csproj", "{346C35DE-D261-4090-A3A9-4D047B60210A}"
+EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.Extensions.Bedrock.MEAI.NetFramework", "src\AWSSDK.Extensions.Bedrock.MEAI\AWSSDK.Extensions.Bedrock.MEAI.NetFramework.csproj", "{4A94F623-0C71-47BD-B927-CB6FA28D33A1}"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AWSSDK.Extensions.Bedrock.MEAI.NetStandard", "src\AWSSDK.Extensions.Bedrock.MEAI\AWSSDK.Extensions.Bedrock.MEAI.NetStandard.csproj", "{B174860A-0D1B-4B7D-9E46-7DBFC9AA5AAB}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AWSSDK.UnitTestUtilities.NetStandard", "..\sdk\test\UnitTests\Custom\AWSSDK.UnitTestUtilities.NetStandard.csproj", "{57891233-C9B6-45C2-AEAD-56F81143C042}"
```

Comment:
Are these changes (adding the generator and test utilities projects) intentional? I don't see any tests for the new projects in the PR.

Reply:
It was intentional but not related to the logging it was to fix the solution building in Visual Studio. For example the `CloudFront.Signers.Tests.NetStandard` project already in the solution had a project reference  to `AWSSDK.UnitTestUtilities.NetStandard` but which has a dependency on `ServiceClientGeneratorLib` and `AWSSDK.CommonTest.NetStandard`. The solution will compile from the command line but in Visual Studio you get errors complaining about the projects not being part of the solution.